app/src/redux/sessions/types.ts

```diff
@@ -320,8 +320,16 @@ export type SessionState = Partial<{
   readonly [robotName: string]: undefined | PerRobotSessionState
 }>
 
+// Unfortunately, the server can sometimes give us numerical slots here, so let's make
+// this a unique and gross type so you can't accidentally check them against a string
+// without converting. Use utils.ts slotNameFromCalibrationSlot and calibrationSlotFromSlotName
+// to convert.
+export type CalibrationLabwareSlot = {
+  readonly __brand: 'UniqueType'
```

Comment:
```suggestion
  readonly __brand: 'CalibrationLabwareSlot'
```
Probably would just give this the brand of the custom type, CalibrationLabwareSlot. If someone else created a branded type with brand `UniqueType`, they'd be identical types.

Reply:
oh right whoops